dali/test/dali_test_config.h

```diff
@@ -19,20 +19,12 @@
 
 namespace dali {
 namespace testing {
-namespace program_options {
 
 /**
- * Parses entry values of C++ executable and stores them internally.
- * You can access these values using functions below.
- */
-void parse_program_options(int argc, const char **argv);
-
-/**
- * Returns value passed using "--dali_extra_path" flag
+ * Returns value to DALI_extra repository
```

Comment:
```suggestion
 * Returns path to DALI test data
```

Reply:
Changed to `path`, but I'd prefer to retain explicit information about DALI_extra repository